### DISCUSSION TITLE
How to remove Tailwind CSS configuration warnings from daisyui-react-vite projects ?

### DISCUSSION THREAD

**Discussion by WackyGem** [tailwindlabs/tailwindcss#11414] [Help] [answered]
I found the following warning while learning the daisyui-react-vite project and reproduced him in my own project, how to make the correct configuration? 

The project address is https://stackblitz.com/edit/daisyui-react-vite/?file=tailwind.config.js .

```
warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
```

**Comment by wongjn** [comment, marked as answer]
As per the warning, you have not used any utility classes in your source files. Once you add a Tailwind utility class to a source file covered by the `content` file globs, the warning should disappear.

**Reply by WackyGem** [reply]
It works! Thank you for your help～
